axoupdater/src/lib.rs

```diff
@@ -275,7 +279,17 @@ impl AxoUpdater {
             }
         };
 
-        Ok(current_version != release.version())
+        // If we're doing "latest" semantics we need to check cur < new
+        // If we're doing "specific" semantics we need to check cur != new
+        let conclusion = match self.version_specifier {
+            UpdateRequest::Latest | UpdateRequest::LatestMaybePrerelease => {
```

Comment:
`LatestMaybePrerelease` is safely covered by this check too?

Reply:
yes, the two are already distinguished in fetch_release, with it changing the filtering